ext/README.md

```diff
@@ -0,0 +1,12 @@
+# ext folder for projects using kubernetes playgound
+
+External projects can create their own folder here.
+The files and folders under `ext` are not visible by git.
+Taking the kites project as an example, this is the recommended
+folder structure:
+
+```text
```

Comment:
We should explain what goes into those dirs

Reply:
done